kasa/iot/iotdevice.py

```diff
@@ -352,6 +352,7 @@ async def _initialize_features(self):
                     name="On since",
                     attribute_getter="on_since",
                     icon="mdi:clock",
+                    category=Feature.Category.Debug,
```

Comment:
We shouldn't do this, but to update the sensorentitydescription to hide them, if we really want to do that.